pyproject.toml

```diff
@@ -22,3 +24,6 @@ ignore_missing_imports = true
 addopts = ["--cov", "validate_pyproject", "--cov-report", "term-missing", "--verbose", "--strict-markers"]
 norecursedirs = ["dist", "build", ".tox"]
 testpaths = ["tests"]
+
+[tool.repo-review]
+ignore = ["PP302", "PP304", "PP305", "PP306", "PP308", "PP309", "PC111", "PC140", "PC170", "PC180", "PC901"]
```

Comment:
`PC111` => We can probably just replace `asottile` with `adamchainz` in the `.pre-commit-config.yaml`.

The rest we can slowly build up with time (except `prettier`, no plans to use nodejs in the CI for a Python project :P)...